lxd/images.go

```diff
@@ -734,6 +734,11 @@ func imagesPost(d *Daemon, r *http.Request) Response {
 			}
 		}
 
+		// Sync the images between each node in the cluster on demand
+		if err := imageSyncBetweenNodes(d, req, info.Fingerprint, imageUpload, post); err != nil {
```

Comment:
we try to avoid one-line function calls + error check in general, please make this two lines